Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020-2021 Hewlett Packard Enterprise Development LP
+# Copyright 2020-2022 Hewlett Packard Enterprise Development LP
```

Comment:
"no other changes..."

Reply:
You successfully caught me in my deception. This was a test and you passed.